compiler/AST/AstToText.cpp

```diff
@@ -1469,7 +1469,16 @@ void AstToText::appendExpr(CallExpr* expr, bool printingType, const char *outer,
         if (i > 1)
           mText += ", ";
 
-        appendExpr(expr->get(i), printingType);
+        if (isSymExpr(expr->get(i)) 
+          && isVarSymbol(toSymExpr(expr->get(i))->symbol()) 
+          && toVarSymbol(toSymExpr(expr->get(i))->symbol())->immediate->const_kind == CONST_KIND_STRING)
```

Comment:
I believe you need a check against if the VarSymbol stores an immediate here, using the `isImmediate()` method.  I think that's why you're getting the internal compiler error - VarSymbols that don't store an immediate will have the immediate field be NULL, causing issues when you look at the const_kind of a NULL

Reply:
Yes, my mistake. I will add ``isImmediate()`` check.